frontend/src/util/preload.ts

```diff
@@ -312,9 +312,7 @@ export const loadSession = async function (secureID: string) {
 			},
 		])
 	} catch (e: any) {
-		const msg = `failed to preload session ${secureID}`
```

Comment:
Is there something else going on here? Like why are we trying to load a session that doesn't exist (or isn't available at the time)?

Reply:
the session is available, but it might not have timeline indicators / network resources, so some of the other graphql calls here may fail.